Transfer/Sources/CreateTransferController.swift

```diff
@@ -280,14 +295,14 @@ extension CreateTransferController: CreateTransferView {
         if let footerView = tableView.footerView(forSection: section.rawValue) as? TransferTableViewFooterView {
             footerView.footerLabel.attributedText = getAttributedFooterText(for: section.rawValue)
         } else {
-            tableView.reloadSections(IndexSet(integer: section.rawValue), with: .none)
+            tableView.reloadSections(IndexSet(integersIn: 0..<section.rawValue + 1), with: .none)
```

Comment:
Why `integersIn: 0..<section.rawValue + 1` ? And you can use `integersIn: 0...section.rawValue` instead